packages/client-web-kit-theme/src/index.css

```
@@ -0,0 +1,48 @@
+ *     ships class strings on its SFCs that any consumer renders.
+ *   - `@vuecs/*` (hoisted to root by npm workspaces) — element class
+ *     maps applied at runtime via the theme manager (e.g.
+ *     `formInput.classes.root = "block w-full rounded-md border ..."`).
+ *     Without these scopes, every `<VCFormInput>` / `<VCButton>` /
+ *     `<VCTable>` renders with `vc-*` defaults and no Tailwind styling.
+ *
+ * Consumer apps add their own `@source` directives for their template
+ * trees + any locally-nested vuecs deps in a per-app CSS entry
+ * (e.g. `apps/client-web/assets/css/tailwind.css`).
+ */
+@source ".";
```

Stylelint (17.11.1)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)